src/controller.cpp

```diff
@@ -132,7 +132,7 @@ void Controller::setPlan(geometry_msgs::Transform current_tf, geometry_msgs::Twi
   if (!track_base_link_enabled_)
   {
     // Add carrot length to plan using goal pose (we assume the last pose contains correct angle)
-    tf2::Transform carrotTF;
+    tf2::Transform carrotTF(tf2::Matrix3x3(1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0, 1.0));
     carrotTF.setOrigin(tf2::Vector3(l_, 0.0, 0.0));
```

Comment:
This line can be combined with the initialisation above right?

Reply:
Nope it's a convenience class, not a message

Reply:
> This line can be combined with the initialisation above right?

Yup that's even neater of course, will do!